api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-18.xml

```diff
@@ -18,7 +18,7 @@
   <concept concept_id="108" retired="false" datatype_id="4" class_id="12" is_set="false" creator="1" date_created="2004-01-01 00:00:00.0" version="" changed_by="1" date_changed="2015-11-05 01:34:22.0"  uuid="108AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="121" retired="false" datatype_id="4" class_id="5" is_set="false" creator="1" date_created="2004-01-01 00:00:00.0" version="" changed_by="1" date_changed="2015-11-05 01:37:39.0"  uuid="121AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="512" retired="false" datatype_id="2" class_id="13" is_set="false" creator="1" date_created="2004-01-01 00:00:00.0" version="596285" changed_by="1" date_changed="2016-04-27 14:18:52.0"  uuid="512AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="924" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2004-07-23 00:00:00.0" version="" changed_by="1" date_changed="2010-10-01 14:33:22.0"  uuid="924AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept concept_id="924" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2004-07-23 00:00:00.0" version="" changed_by="1" date_changed="2017-12-14 12:34:09.0"  uuid="924AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```

Comment:
What was changed on the above line to necessitate a date_changed change?

Reply:
Nothing. Why does the date change happen without any perceptible changes?

Reply:
So, can we avoid such changes of only the date_changed?

Reply:
Yes sure, I am already on it. Will push a commit in a few mins :)